### PR TITLE
Payment confirmation

### PR DESCRIPTION
**Pull request by almoxxx**
# Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Description

Vendor Overview - qrcode view and credit view added, as well as integration of the demo api call.
Dependend on backend api changes --> qrcode url is partly hardcoded so far, will be needed to be changed and stored in pinia

# Checklist:

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -46,6 +46,16 @@ const router = createRouter({
...
+    },
+    {
+      path: '/paymentconfirmation',
+      name: 'Payment Confirmation',
+      component: () => import('../views/PaymentConfirmation.vue')
+    },
+    {
+      path: '/vendoroverview',
+      name: 'Vendor Overview',
+      component: () => import('../views/VendorOverview.vue')
```
Comment (nanu-c) [inline src/router/index.ts:58]: We have `/vendoroverview` and `/vendor/overview`. Can you remove this one again?

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: src/router/index.ts (hunk 1 above)
Comment (almoxxx) [reply src/router/index.ts:48]: I removed that part and also removed the redundant file which I forgot earlier. Can you please check if it is alright now? :)

File: src/views/PaymentConfirmation.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+let time = ref('not')
+function currentTime() {
+  const now = new Date()
+  time.value = `${now.getHours()}:${now.getMinutes()}:${now.getSeconds()}`
+}
+
+function UpdateTime() {
+  setTimeout(() => {
+    console.log('hello')
```
Comment (lebe1) [inline src/views/PaymentConfirmation.vue:36]: Please delete this logging info. I have a funny hello counting almost twice a second in my console ;)

File: src/stores/vendor.ts
Comment (lebe1) [inline src/stores/vendor.ts:1]: Also is this file necessary?
Since you are not using it at all, for now I guess not but since credit is a variable which probably will be used across the whole application, I assume it might be safer to define a Pinisstore at least for this one.
But @nanu-c should have a thought on this one too

**Review by lebe1 — COMMENTED** [review]
Two more points I've found. One really small logging line to be deleted and another point to consider if a PiniaStore should not be defined.

**Review by nanu-c — APPROVED** [review]
